pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -276,7 +300,13 @@ stdenv.mkDerivation (finalAttrs: {
     platforms = platforms.unix;
   }
   # toolsOnly: Does not have qemu-kvm and there's no main support tool
-  // lib.optionalAttrs (!toolsOnly) {
+  # userOnly: There's one qemu-<arch> for every architecture
+  // lib.optionalAttrs (!toolsOnly && !userOnly) {
     mainProgram = "qemu-kvm";
+  }
+  # userOnly: Only meaningful on linux
+  // lib.optionalAttrs userOnly {
+    platforms = lib.platforms.linux;
```

Comment:
QEMU can also emulate BSD from BSD.

Reply:
I'm unsure what that implies. I tried building `pkgsCross.x86_64-{open,free,net}bsd.qemu-user`, but just got stuck on various missing dependencies. I could set this to `with lib.platforms; [ linux freebsd openbsd netbsd ];` or `lib.platforms.unix`, but I cant test it… I did at least update the comment.

Reply:
I see. Set to `linux ++ freebsd ++ openbsd ++ netbsd` now. I think updating the hostCpuTargets defaults to conditionally include `*-bsd-user` is not worth it, since this is a theoretic exercise.